shared/protocol/src/main/java/io/pravega/shared/protocol/netty/ExceptionLoggingHandler.java

```diff
@@ -36,7 +36,7 @@ public void channelRead(ChannelHandlerContext ctx, Object msg) throws Exception
         try {
             super.channelRead(ctx, msg);
         } catch (Exception e) {
-            log.error("Uncaught exception on connection " + connectionName, e);
+            log.error("Uncaught exception observed during channel read on connection " + connectionName, e);
```

Comment:
Please, use `{}` instead of string concatenation for `connectionName` parameter.

Reply:
done.